src/hotspot/share/code/compiledIC.cpp

```diff
@@ -734,10 +709,10 @@ void CompiledDirectStaticCall::print() {
     tty->print("clean");
   } else if (is_call_to_compiled()) {
     tty->print("compiled");
-  } else if (is_call_to_far()) {
-    tty->print("far");
   } else if (is_call_to_interpreted()) {
     tty->print("interpreted");
+  } else {
+    tty->print("unknown");
```

Comment:
We can probably split this cleanup into the minor issue, your call. The benefit of separate issue: backportability.

Reply:
Reverted and filed https://bugs.openjdk.java.net/browse/JDK-8265013